components/subject-resources.tsx

```
@@ -4,25 +4,31 @@
 import { useState } from 'react'
 import { Resource, ResourceType } from '@prisma/client'
 import Link from 'next/link'
-import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card'
+import { Card } from '@/components/ui/card'
 import { Badge } from '@/components/ui/badge'
 import { BookOpen, Book, FileText } from 'lucide-react'
 import { Button } from '@/components/ui/button'
 import Image from 'next/image'
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs-variants";
```

ESLint

[error] 'Tabs' is defined but never used.

(@typescript-eslint/no-unused-vars)